docs/reference/commandline/ps.md

```diff
@@ -174,9 +174,19 @@ Any of these events result in a `137` status:
 
 #### status
 
-The `status` filter matches containers by status. You can filter using
-`created`, `restarting`, `running`, `removing`, `paused`, `exited` and `dead`. For example,
-to filter for `running` containers:
+The `status` filter matches containers by status. The possible values for the container status are:
+
+| Status                | Description                                                                                                                          |
+|:----------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| `created`             | A container that has never been started. |
```

Comment:
Could you format the table to have the borders aligned? I think we format all our tables like that (and I know for sure my IDE will complain about them not being formatted ... but that's on my machine 😂 )

Reply:
I tried using Prettier.  Is this the formatting you are after?

Reply:
Yup

Well, mostly, but I have an open ticket in JetBrain's issue tracker for that 😞 https://youtrack.jetbrains.com/issue/IDEA-332005/Markdown-table-headers-formatting-does-not-match-GFM-spec